env-data.sh

```diff
@@ -144,3 +144,25 @@ fi
 if [ ! -z "$POSTGRES_DB" ]; then
 	POSTGRES_DBNAME=${POSTGRES_DB}
 fi
+
+
+# usable function definitions
+function restart_postgres {
+PID=`cat ${PG_PID}`
+kill -TERM ${PID}
+
+# Wait for background postgres main process to exit
+while [[ "$(ls -A ${PG_PID} 2>/dev/null)" ]]; do
+  sleep 1
+done
+
+# Brought postgres back up again
+source /env-data.sh
+su - postgres -c "${POSTGRES} -D ${DATADIR} -c config_file=${CONF} ${LOCALONLY} &"
+
+# wait for postgres to come up
+until su - postgres -c "psql -l"; do
+  sleep 1
+done
+echo "postgres ready"
+}
```

Comment:
@lucernae Why do we have this function in env-data.sh and is it used anywhere ?

Reply:
so that anyone can import the script and use it from their own db init script.
Like what I used here:

https://github.com/kartoza/fbf-project/pull/203/files#diff-247ddbdc4faacc35e5592890dbd96a25R30

It does really help when setting up something and you need to restart postgres before running the next script/SQL

Reply:
I understand @lucernae